Documentation/Functions/If.rst

```diff
@@ -20,6 +20,9 @@ There is no else property available. The "else" branch of an "if" statement is a
 missing feature. You can implement a workaround by a logic based on the
 :ref:`stdwrap-override-conditions`.
 
+Simple "if empty use different value" conditions can be built with the
+:ref:`TypoScript // (double slash) <data-type-gettext>` fallback operator.
```

Comment:
I suggest to add an anchor to the section and use it here, so people don't have to search for the section. What do you think?